2024-08-09 03:55:06,430 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.34.59.2:44661'
2024-08-09 03:55:06,434 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.34.59.2:34147'
2024-08-09 03:55:07,067 - distributed.diskutils - INFO - Found stale lock file and directory '/tmp/dask-scratch-space/worker-1prkn4bu', purging
2024-08-09 03:55:07,068 - distributed.diskutils - INFO - Found stale lock file and directory '/tmp/dask-scratch-space/worker-x_okzye3', purging
2024-08-09 03:55:07,576 - distributed.worker - INFO -       Start worker at:     tcp://10.34.59.2:34100
2024-08-09 03:55:07,576 - distributed.worker - INFO -          Listening to:     tcp://10.34.59.2:34100
2024-08-09 03:55:07,576 - distributed.worker - INFO -           Worker name:           SLURMCluster-0-1
2024-08-09 03:55:07,576 - distributed.worker - INFO -       Start worker at:     tcp://10.34.59.2:41185
2024-08-09 03:55:07,576 - distributed.worker - INFO -          dashboard at:           10.34.59.2:41116
2024-08-09 03:55:07,576 - distributed.worker - INFO - Waiting to connect to:     tcp://10.34.59.1:35887
2024-08-09 03:55:07,576 - distributed.worker - INFO -          Listening to:     tcp://10.34.59.2:41185
2024-08-09 03:55:07,576 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 03:55:07,576 - distributed.worker - INFO -           Worker name:           SLURMCluster-0-0
2024-08-09 03:55:07,576 - distributed.worker - INFO -               Threads:                          1
2024-08-09 03:55:07,576 - distributed.worker - INFO -          dashboard at:           10.34.59.2:39818
2024-08-09 03:55:07,576 - distributed.worker - INFO -                Memory:                   1.86 GiB
2024-08-09 03:55:07,576 - distributed.worker - INFO - Waiting to connect to:     tcp://10.34.59.1:35887
2024-08-09 03:55:07,576 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-b_thirwq
2024-08-09 03:55:07,576 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 03:55:07,576 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 03:55:07,576 - distributed.worker - INFO -               Threads:                          1
2024-08-09 03:55:07,576 - distributed.worker - INFO -                Memory:                   1.86 GiB
2024-08-09 03:55:07,577 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-xn2wd57p
2024-08-09 03:55:07,577 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 03:55:07,978 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-08-09 03:55:07,979 - distributed.worker - INFO -         Registered to:     tcp://10.34.59.1:35887
2024-08-09 03:55:07,979 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-08-09 03:55:07,979 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 03:55:07,979 - distributed.core - INFO - Starting established connection to tcp://10.34.59.1:35887
2024-08-09 03:55:07,980 - distributed.worker - INFO -         Registered to:     tcp://10.34.59.1:35887
2024-08-09 03:55:07,980 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 03:55:07,980 - distributed.core - INFO - Starting established connection to tcp://10.34.59.1:35887
2024-08-09 03:55:23,330 - distributed.worker - INFO - Stopping worker at tcp://10.34.59.2:41185. Reason: scheduler-close
2024-08-09 03:55:23,330 - distributed.worker - INFO - Stopping worker at tcp://10.34.59.2:34100. Reason: scheduler-close
slurmstepd: error: *** JOB 2328816 ON cl-node002 CANCELLED AT 2024-08-09T03:55:23 ***
2024-08-09 03:55:23,331 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Worker->Scheduler local=tcp://10.34.59.2:34726 remote=tcp://10.34.59.1:35887>
Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 297, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/tornado/gen.py", line 767, in run
    value = future.result()
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 307, in write
    convert_stream_closed_error(self, e)
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Worker->Scheduler local=tcp://10.34.59.2:34726 remote=tcp://10.34.59.1:35887>: Stream is closed
